tools/visual-pipeline-and-platform-evaluation-tool/Makefile

```diff
@@ -13,6 +13,17 @@ SHELL := bash -eu -o pipefail
 PROJECT_NAME := vippet
 PROJECT_DIR := $(shell pwd)
 
+# Release flag (true/false)
+RELEASE ?= false
+
+# Code Versions
+VERSION := v1.2
+GIT_COMMIT := $(shell git rev-parse --short HEAD)
+DOCKER_TAG := $(VERSION)
+ifeq ($(RELEASE),false)
+    DOCKER_TAG := $(VERSION)-$(GIT_COMMIT)
+endif
```

Comment:
Maybe we should introduce a VERSION file to keep the version in it?
I’m not saying we need to bump it with every PR right away, but it just seems nicer to me than keeping it in the `Makefile`.